pkg/config/config_test.go

```diff
@@ -96,6 +102,57 @@ var _ = Describe("Config", func() {
 		Expect(cfg.Trunk).To(BeEquivalentTo([]int{5, 19, 20, 21, 22, 23, 55, 101, 102, 103}))
 
 	})
+	It("Assuming correct config file - static bridge config", func() {
+		data := []byte(`{
+		"name": "mynet",
+		"type": "accelerated-bridge",
+		"deviceID": "0000:af:06.1",
+		"bridge": "bridge1, bridge2"
+		}`)
+		mockNetlink := &mocks.Netlink{}
+		mockNetlink.On("LinkByName", mock.Anything).Return(
+			&netlink.Device{LinkAttrs: netlink.LinkAttrs{Name: existingPF, MasterIndex: 2}}, nil)
+		mockNetlink.On("LinkByIndex", 2).Return(
```

Comment:
This  adds an additional call to mockNetlink 

with current code, the call added to mockNetlink on L30 will not be called right ?
if i run mockNetlink.AssertExpectations() will it fail ?

i dont think its a good practice to add a mock call if its not going to be used.

how about splitting to two contexts ? one for "regular parse conf" flows
and another for bridge auto detection feature, where you do fine grained mock in each test

if i am totaly wrong, then disregard this comment :)

Reply:
you are right, I refactored all tests for config package (in a separate commit)